CRM/Contact/Import/Form/MapField.php

```diff
@@ -717,39 +717,15 @@ public function formatCustomFieldName(&$fields) {
    * @return \CRM_Contact_Import_Parser_Contact
    */
   public function submit($params, $mapperKeys) {
-    $mapper = $mapperKeysMain = array();
+    $mapper = $mapperKeysMain = $locations = array();
+    $parserParameters = CRM_Contact_Import_Parser_Contact::getParameterForParser($this->_columnCount);
```

Comment:
currently this just gets a base populated array - later I would like to move the whole derivation of $parserParameters into there & share between the 2 functions -but I feel that is too much for one pr